compiler/src/typed/typedtreeIter.re

```diff
@@ -122,7 +122,7 @@ module MakeIterator =
     Iter.leave_data_declaration(decl);
   }
 
-  /* FIXME: These two functions are gross */
+  // TODO(#1505): Refactor this function
```

Comment:
@peblair What's gross about them? I feel like we can just delete this comment and close the issue.

Reply:
Going to just close and reopen if we feel it needs refactoring.